### DISCUSSION TITLE
Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship. Check out our guide on how to contribute to open-source projects to learn how you can make a meaningful impact! Keep up the great work and you’ll be a perfect fit in no time!

### DISCUSSION THREAD

**Discussion by redoC-A2k** [community/community#51994] [GitHub Education]
I am a active contributor on github , Please fix this ... I am unable to apply closing date is very near (15 april)
First it took me very long to reverify my identity about 7-8 times my application rejected and now this issue of contribution , even though there is nothing mentioned about it in elligiblity or FAQ.
Please Please help , I am in my pre-final year and in search of such open source programs

**Comment by jge162** [comment]
Check guide on how to contribute to open-source projects.
